### PR TITLE
integration-tests: Add role binding test suite

### PR DESCRIPTION
**Pull request by alejovicu**
Tests added (add RB by user, group and service account)
https://jira.coreos.com/browse/CONSOLE-396

### CODE REVIEW

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by ggreer** [comment]
jenkins rebuild

**Comment by ggreer** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/integration-tests/views/role-bindings.view.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+
+const BROWSER_TIMEOUT = 15000;
+
+export const titleLbl = $('#resource-title');
+export const createBtn = $('#yaml-create');
```
Comment (jhadvig) [inline frontend/integration-tests/views/role-bindings.view.ts]: Both of these const are already defined in `crud.view.ts`

**Comment by jhadvig** [comment]
Just a one comment. Otherwise good to merge :+1:

**Pull request closed without merging**